src/solvers/dgsem_p4est/dg_2d.jl

```diff
@@ -177,39 +247,54 @@ function prolong2mortars!(cache, u,
                           mortar_l2::LobattoLegendreMortarL2,
                           surface_integral, dg::DGSEM)
   @unpack element_ids, node_indices = cache.mortars
-
-  size_ = (nnodes(dg), nnodes(dg))
+  idx_one = 1
+  idx_end = nnodes(dg)
 
   @threaded for mortar in eachmortar(dg, cache)
+    # Copy solution data from the small elements on a case-by-case basis to get
+    # the correct face and orientation.
     small_indices = node_indices[1, mortar]
-    large_indices = node_indices[2, mortar]
 
-    # Copy solution small to small
+    i_small_start, i_small_step = index_to_start_step(
+      small_indices[1], idx_one, idx_end)
+    j_small_start, j_small_step = index_to_start_step(
+      small_indices[2], idx_one, idx_end)
+
     for pos in 1:2
```

Comment:
```suggestion
    for position in 1:2
```
I don't know why this was abbreviated in the first place